SwiftWisdom.xcodeproj/project.pbxproj

```diff
@@ -575,12 +587,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 80232BC71BF2EE2E00818B6E /* Build configuration list for PBXNativeTarget "SwiftWisdom" */;
 			buildPhases = (
+				21125417B2D0E6F75BB64B48 /* [CP] Check Pods Manifest.lock */,
 				C870EFFC951BFB3182EF5126 /* [CP] Check Pods Manifest.lock */,
 				80232B9B1BF2EE2E00818B6E /* Sources */,
 				80232B9C1BF2EE2E00818B6E /* Frameworks */,
 				80232B9D1BF2EE2E00818B6E /* Resources */,
 				9A1B03B8C9D2B525E066F8AA /* [CP] Embed Pods Frameworks */,
 				D7FE8601F9C058C38C034313 /* [CP] Copy Pods Resources */,
+				7C9A62994AB6B82BDD09F259 /* 📦 Embed Pods Frameworks */,
+				CE9C94B81093D2BE1BA7D4FC /* 📦 Copy Pods Resources */,
```

Comment:
Yeah I noticed I was on 1.0, after pod install, then upgraded. And it looks like it duplicated stuff. I'll fix that.